app/helpers/automated_tests_helper.rb

```diff
@@ -1,6 +1,34 @@
 # Helper methods for Testing Framework forms
 module AutomatedTestsHelper
 
+  def create_test_repo(assignment)
+    # Create the automated test repository
+    unless File.exists?(MarkusConfigurator.markus_config_automated_tests_repository)
```

Comment:
`File.exists?` is deprecated in favor of `File.exist?`.<br>Line is too long. [84/80]